source/connect-atlas-cli.txt

```diff
@@ -13,8 +13,8 @@ Connect from the {+atlas-cli+}
 Select a Connection Method
 --------------------------
 
-When you connect to |service| from the {+atlas-cli+}, you can authenticate with
-one of the following commands:
+When you connect to an existing |service| account from the 
```

Comment:
adding a comment here just because the connect page recommends using --skipConfig, but the flag will be deprecated. 

they don't need to ask to skip config, the new behavior is:
1) if they are logging in for the first time and have several projects/orgs, we'll ask them to select one
2) if they are logging in again, we'll not ask anything unless the project that was configured is invalid

Reply:
Also, on the important warning that we tell customers to delete the default profile, we can also add another option which would be login with a new profile by running atlas login -P newProfile